xml/System.Reflection.Metadata.Ecma335/MetadataRootBuilder.xml

```diff
@@ -146,8 +154,8 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Determines if basic validation of metadata tables should be suppressed. The validation verifies that entries in the tables were added in order required by the ECMA specification. It does not enforce all specification requirements on metadata tables.</summary>
+        <value><see langword="true" /> to suppress basic validation of metadata tables; <see langword="false" /> otherwise.</value>
```

Comment:
Standard wording is reversed 😄 
```suggestion
        <value><see langword="true" /> to suppress basic validation of metadata tables; otherwise <see langword="false" />.</value>
```